testnets/okp4testnet/chain.json

```diff
@@ -97,6 +97,14 @@
         "address": "91.211.251.232:28656"
       }
     ]
+  },
```

Comment:
Same as your other PR, I think this line needs to be removed and then your check will pass.